build/opnsense.json

```diff
@@ -58,19 +53,17 @@
         [ "modifyvm", "{{.Name}}", "--memory", "1024" ],
         [ "modifyvm", "{{.Name}}", "--cpus", "1" ],
         [ "modifyvm", "{{.Name}}", "--boot1", "disk" ],
-        [ "modifyvm", "{{.Name}}", "--boot2", "dvd" ],
         [ "modifyvm", "{{.Name}}", "--audio", "none" ],
         [ "modifyvm", "{{.Name}}", "--usb", "off" ]
       ],
       "post_shutdown_delay": "30s",
       "vboxmanage_post": [
-        [ "modifyvm", "{{.Name}}", "--nic1", "intnet" ],
+        [ "modifyvm", "{{.Name}}", "--nic1", "nat" ],
```

Comment:
I strongly tend to not go this way, since i want to use OPNsense here as its used in production with a encapsulated LAN network, see the statements above